features/src/main/feature/camel-features.xml

```diff
@@ -765,6 +765,22 @@
         <bundle dependency='true'>mvn:commons-io/commons-io/2.15.1</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-exec/${project.version}</bundle>
     </feature>
+    <feature name='camel-elytron' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version='${project.version}'>camel-http</feature>
+        <feature>jakarta-servlet</feature>
+        <bundle dependency='true'>mvn:org.apache.camel.karaf/camel-undertow/${project.version}</bundle>
+        <bundle dependency='true'>mvn:io.undertow/undertow-core/${undertow-version}</bundle>
+        <bundle dependency='true'>mvn:io.undertow/undertow-servlet/${undertow-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:org.wildfly.common/wildfly-common/1.5.4.Final$Export-Package=*;version=1.5.4.Final</bundle>
+        <bundle dependency='true'>mvn:org.jboss.xnio/xnio-api/3.8.9.Final</bundle>
+        <bundle dependency='true'>mvn:org.jboss.xnio/xnio-nio/3.8.9.Final</bundle>
+        <bundle dependency='true'>wrap:mvn:org.jboss.threads/jboss-threads/2.3.3.Final</bundle>
```

Comment:
According to the command `dependency:tree` the expected version is `3.5.0.Final`

Reply:
ok fixed